ReactWindows/ReactNative.Shared/Modules/Network/NetworkingModule.cs

```diff
@@ -450,12 +500,20 @@ private static void TranslateHeaders(JObject headerData, IDictionary<string, str
 
         private static IHttpClient CreateDefaultHttpClient()
         {
+#if WINDOWS_UWP
             return new DefaultHttpClient(
                 new HttpClient(
                     new HttpBaseProtocolFilter
```

Comment:
type alias HttpBaseProtocolFilter and WebRequestHandler to consolidate #ifs for better readability.